fdbclient/IClientApi.h

```diff
@@ -79,6 +79,8 @@ class ITransaction {
 
 	virtual ThreadFuture<Void> commit() = 0;
 	virtual Version getCommittedVersion() = 0;
+	virtual std::string getVersionVector() = 0;
```

Comment:
why returning a `string` instead of a version vector data type?

Reply:
I used this method to dump the version vector in workload actors in case of an error (like, data mismatch). A string was all I needed in that context.

I can change it to return the VersionVector if you think that is more appropriate.

Reply:
As an API, I'd prefer to the VersionVector type, so that the caller of this API doesn't need to deserialize the data.

Reply:
I did not see `getVersionVector` used, can it be added to a trace?

Reply:
I used this API while investigating the database corruption issue. Dropped it after that.

Reply:
Sorry, where do you want me to add that trace?

Reply:
I do not have a place to add it, it just struck me that there was a new API introduced unused. Sometimes that can lead to dead code. Maybe you could add `if (info.debugID.present())` to conditionally use it.